### ISSUE TITLE
[QUESTION] How to style past date without disabling them ?

### ISSUE THREAD

**Issue by mrleblanc101** [nathanreyes/v-calendar#859]
Hi, I would like to lighten the past dates on the date-picker, but I do not want to disabled them.
Is this possible ? Currently there is no class identifying past date unless I add `:min-date="new Date()"`.
This add a `is-disabled` class which lighten the text color, exactly like I want, but now I can't select them.
Maybe there should be a `is-past` class by default on past dates ?

**Comment by Helcostr** [comment]
U can do an attribute + styleClass + date expression.

**Comment by mrleblanc101** [comment]
Could you give an example ? I'm not sure how to get past date with expression.
Also this seems pretty complicated for such a simple feature.
The plugin should have class for past date, even if the default theme does not style them.

**Comment by Helcostr** [comment]
The only reason why I suggest this is because I was just studying the attributes prop, and I already worked out how to do it in my head.

Here.

```css
  :deep(.pastDateBG) {
    display: none;
  }
  :deep(.pastDate){
    color:var(--gray-500);
  }
```

```javascript
{
  key:'past',
  highlight:{
    class:'pastDateBG',
    contentClass:'pastDate'
  },
  dates:{
    end: new Date()
  }
}
```

**Comment by mrleblanc101** [comment]
Works well, thanks !
Still think it's a bit of a hack. It's not clear you can specify an `end` without a `start`

**Comment by Helcostr** [comment]
> A null value denotes an infinite start or end date.
> Optionally, if using null dates, you can omit them entirely.
> Thus, an empty object is a valid date expression...
https://vcalendar.io/date-expressions.html

**Comment by mrleblanc101** [comment]
Yes I understand that now, but you need to dig deep in the doc to figure this out. Styling past-date is a very common feature for calendar/date-picker so that's why I propose a default class on past date
Just like today has `is-today`, past date could have `is-past` and it would make everything much simpler, no ?
